libs/core/src/lib/resizable-card-layout/resizable-card-item/resizable-card-item.component.scss

```diff
@@ -0,0 +1,2255 @@
+/*!
+ * Fundamental Library Styles v0.17.0-rc.30
+ * Copyright (c) 2021 SAP SE or an SAP affiliate company.
+ * Licensed under Apache License 2.0 (https://github.com/SAP/fundamental-styles/blob/main/LICENSE)
+ */
+ @charset "UTF-8";
+ [class*=sap-icon] {
+   font-size: .875rem;
```

Comment:
Should `@import` the fundamental-styles icon.css here.

Reply:
same as above

Reply:
There is no `resizable-card.css` in `0.17.0`, let's keep it as it is